lib/pages/timetable/service/uni_event_provider.dart

```diff
@@ -407,3 +424,9 @@ class UniEventProvider extends EventProvider<UniEventInstance>
     }
   }
 }
+
+extension GoogleCalendarEventExtension on g_cal.Event {
+  String myToString() {
+    return summary;
+  }
+}
```

Comment:
I don't understand what this is for and why it is here. First "myToString" sounds like something you'd use in a codelab, not in real code, since it doesn't convey any information about what this is supposed to do. Second, AFA I can see you're only using this in some print statements, can't you just use event.summary directly?